admin/js/editor.js

```diff
@@ -44,17 +44,18 @@ wp.domReady(() => {
     ]),
   );
 
-  ['core/group'].forEach(block => {
-    registerBlockStyle(block, {
-      name: 'space-evenly',
-      label: __('Space evenly', 'planet4-blocks-backend'),
-    });
+  registerBlockStyle('core/group', {
+    name: 'space-evenly',
+    label: __('Space evenly', 'planet4-blocks-backend'),
   });
 
-  ['core/paragraph'].forEach(block => {
-    registerBlockStyle(block, {
-      name: 'roboto-font-family',
-      label: __('Roboto font family', 'planet4-blocks-backend'),
-    });
+  registerBlockStyle('core/paragraph', {
```

Comment:
The same [from here](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/848/files#r870239342)